workloads/router-perf-v2/common.sh

```diff
@@ -51,7 +52,15 @@ get_scenario(){
 deploy_infra(){
   log "Deploying benchmark infrastructure"
   envsubst < ${INFRA_TEMPLATE} > ${INFRA_CONFIG}
-  ${ENGINE} run --rm -v $(pwd)/templates:/templates:z -v ${KUBECONFIG}:/root/.kube/config:z -v $(pwd)/${INFRA_CONFIG}:/http-perf.yml:z ${KUBE_BURNER_IMAGE} init -c http-perf.yml --uuid=${UUID}
+  if [[ ${ENGINE} == "local" ]]; then
+    log "Downloading kube-burner binary"
+    curl ${KUBE_BURNER_RELEASE_URL}
+    log "Extracting kube-burner binary"
+    tar -xvzf kube-burner.tar.gz ; rm -f kube-burner.tar.gz
```

Comment:
What about replacing this lines by something like this?

```shell
curl -LsS ${KUBE_BURNER_RELEASE_URL} | tar xz
```

Reply:
Thanks Raul. I tried that and I get ```gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now``` 
But I added `-LsS` , updating commit.

Reply:
found a mistake. Fixed it. Your suggestion works.